osu.Game/OsuGameBase.cs

```diff
@@ -64,6 +64,8 @@ public class OsuGameBase : Framework.Game, ICanAcceptFiles
 
         protected FileStore FileStore;
 
+        protected GlobalActionContainer GlobalBinding;
```

Comment:
this should be renamed or simply not exposed. we don't expose things for tests any more. please use `ChildrenOfType` instead.

Reply:
I see, will take this into account in the future.